scenes/Restaurant/Customer.gd

```diff
@@ -165,6 +161,11 @@ func _update_customer_properties() -> void:
 			get_node(node_path).material.set_shader_param(shader_param, _customer_def[key])
 	$Body.update()
 	visible = true
+	
+	if Engine.is_editor_hint():
+		# When we're using this as a tool in the editor, we only update the graphics
```

Comment:
Maybe make this comment clearer. "We only update the graphics" is sort of a confusing sentence fragment

Reply:
Rephrased to make it clear what we're skipping and (implicitly) why we're skipping it